releasenotes/notes/feat-add-dd-record-exception-033fd0436dfd2723.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    tracing: tracing: Introduces a record_exception method that adds an exception to a Span as a span event.
+    Refer to [Span.record_exception](https://ddtrace.readthedocs.io/en/stable/api.html#ddtrace.trace.Span.record_exception) 
```

Comment:
Can we use a rst link here

Reply:
I would say no, as it is not a .rst and the api doc is auto generated.